content/help/ops/traffic-management/proxy-cmd/index.md

```diff
@@ -278,6 +278,16 @@ one route that matches on everything. This route tells Envoy to send the request
         }
     ]
     {{< /text >}}
+    
+1. To see the endpoints currently available for the cluster use the proxy-config endpoints command.
+
+    {{< text bash json >}}
+    $ istioctl proxy-config endpoints productpage-v1-6c886ff494-7vxhs --cluster=outbound|9080||reviews.default.svc.cluster.local
```

Comment:
Nit: Could you remove the `=` here

Reply:
Done. But wondering what's difference about have or no `=`, all can work.

Reply:
Just consistency with the rest of the examples

Reply:
ok :-)